protocol/x/prices/keeper/market_param.go

```diff
@@ -48,6 +49,20 @@ func (k Keeper) ModifyMarketParam(
 	marketParamStore := k.getMarketParamStore(ctx)
 	b := k.cdc.MustMarshal(&updatedMarketParam)
 	marketParamStore.Set(lib.Uint32ToKey(updatedMarketParam.Id), b)
+	
+	// if the market pair has been changed, we need to update the in-memory market pair cache
+	if existingParam.Pair != updatedMarketParam.Pair {
+		// remove the old cache entry
+		k.currencyPairIDCache.Remove(uint64(existingParam.Id))
+
+		// add the new cache entry
+		cp, err := slinky.MarketPairToCurrencyPair(updatedMarketParam.Pair)
+		if err == nil {
+			k.currencyPairIDCache.AddCurrencyPair(uint64(updatedMarketParam.Id), cp.String())
```

Comment:
didn't we decide on using the same format for currencyPair.string() and marketParam.Pair? Shouldn't these two always be equal

Reply:
currency-pairs in slinky are marshalled as `BASE/QUOTE` whereas it looks like DYDX handles them as `BASE-QUOTE` hence the conversion. The same conversion is performed in the slinky-price-daemon as well